Docs deprecate workflow (#48261)
ray-project/ray: doc/source/workflows/index.rst

.. warning::

>   Ray Workflows is not currently maintained and will be deprecated in the future. Please use higher-level alternatives, such as Airflow or Prefect, to orchestrate your workflow. 

Reviewer: ```suggestion
  Ray Workflows isn't currently maintained and will be deprecated. Use higher-level alternatives, such as Airflow or Prefect, to orchestrate your workflow. 
```
